jumanji/environments/packing/bin_pack/generator.py

```diff
@@ -1312,20 +1140,179 @@ def __init__(
             split_num_same_items,
             container_dims,
         )
+        self.is_rotation_allowed = is_rotation_allowed
+        self.is_value_based = is_value_based
+        self.mean_value = mean_value
+        self.standard_deviation_value = standard_deviation_value
+        if self.is_value_based and (
+            mean_value is None or standard_deviation_value is None
+        ):
+            std_name = "standard_deviation_value"
+            raise ValueError(
+                "Value Based generator not provided with"
+                + f"{'mean_value attribute' if mean_value is None else ''}"
+                + f"{f'{std_name} attribute' if standard_deviation_value is None else ''}"
+            )
 
-    def _generate_solved_instance(self, key: chex.PRNGKey) -> State:
+    def _generate_value_based_solved_instance(self, key: chex.PRNGKey) -> State:
+        """Generate the random instance with half of all items correctly packed (the higher value
+        half). The other half of the generated items remain unpacked.
+
+        The first half of items are generated by splitting the container in the same way as the
+        inherited RandomGenerator class, but with less than half the max_num_items. Values are
+        randomly generated, doubled and assigned to these items. These items are
```

Comment:
Comment is out of date, they aren't doubled anymore.

Is the rest of this a copy and paste of what was in the original Value based generator?

Reply:
The values are not doubled, the items are duplicated. The values are value + total value of original items (though it needs to be explained clearer than this).

Reply:
I modified the comment but feel free to add a suggestion here.

Reply:
Great thanks!